jwst/datamodels/cube.py

```diff
@@ -44,43 +44,3 @@ class CubeModel(model_base.DataModel):
         The variance due to read noise array.  3-D
     """
     schema_url = "cube.schema.yaml"
-
-    def __init__(self, init=None, data=None, dq=None, err=None, zeroframe=None,
-                 relsens=None, int_times=None, area=None, wavelength=None,
-                 var_poisson=None, var_rnoise=None, **kwargs):
-      
-        super(CubeModel, self).__init__(init=init, **kwargs)
-
-        if data is not None:
-            self.data = data
-
-        if dq is not None:
-            self.dq = dq
-
-        if err is not None:
-            self.err = err
-
-        if zeroframe is not None:
-            self.zeroframe = zeroframe
-
-        if relsens is not None:
-            self.relsens = relsens
-
-        if int_times is not None:
-            self.int_times = int_times
-
-        if area is not None:
-            self.area = area
-
-        if wavelength is not None:
-            self.wavelength = wavelength
-
-        if var_poisson is not None:
-            self.var_poisson = var_poisson
-
-        if var_rnoise is not None:
-            self.var_rnoise = var_rnoise
-   
-        # Implicitly create arrays
-        self.dq = self.dq
```

Comment:
So how or where is this functionality now handled, so that the DQ and ERR arrays get "inflated" even when they aren't present in the incoming data?

Reply:
Looks like `CubeModel` is now subclasssed from `ImageModel`, which means it gets the array inflation goodies in `ImageModel`, but of course has its own schema, so dimensions of the attributes are not inherited from `ImageModel`

Reply:
Though I see that there isn't a line in the `__init__` method that has
```
super(CubeModel, self).__init__(init=init, **kwargs)
```

Reply:
In fact there isn't an `__init__` method.  Should there be?  `ImageModel` which subclasses `DataModel` has one.  I would think we would need one here too?